tests/jballerina-integration-test/src/test/resources/testng.xml

```diff
@@ -71,6 +71,7 @@
             <class name="org.ballerinalang.test.service.http.sample.SslProtocolTest"/>
             <class name="org.ballerinalang.test.service.http.sample.SerializeComplexXmlTestCase"/>
             <class name="org.ballerinalang.test.service.http.sample.HTTPListenerMethodTestCase"/>
+            <class name="org.ballerinalang.test.service.http.sample.HTTPCookiesTestCase"/>
```

Comment:
Shall we also run the same test-cases for Retry and CB configurations. Also, let's make sure if need cookie config for FB and LB clients.